src/web/components/ClientSideCompletion/ClientSideCompletion.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { faker } from '@faker-js/faker';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ClientSideCompletion } from './ClientSideCompletion';
+
+function createKeypairFake(disabled: boolean) {
+  return {
+    created: faker.date.recent(),
+    createdString: faker.date.recent().toLocaleTimeString(),
+    disabled,
+    publicKey: faker.string.hexadecimal(),
+    siteId: faker.number.int(),
+    subscriptionId: faker.string.alpha(),
+  };
+}
+
+const meta: Meta<typeof ClientSideCompletion> = {
+  component: ClientSideCompletion,
+};
+
+export default meta;
+type Story = StoryObj<typeof ClientSideCompletion>;
+
+export const ReadyToCSTG: Story = {
+  args: {
+    domainNames: ['example.com'],
+    keyPairData: [createKeypairFake(false)],
+  },
+  storyName: 'Ready to CSTG (renders empty - nothing to say!)',
+  name: 'Ready to CSTG (blank)',
```

Comment:
I know the Ready to CSTG tab is supposed to be blank, but it's a little confusing if it is broken or blank.  I like the text under storyName here if it could be displayed

Reply:
It's a little weird but it's working as intended because the component is rendering itself as it should.  We'd have to create a wrapper component to show a message.  Hopefully the (blank) in the navigation is semi-clear.